rocAL/rocAL_pybind/examples/rocAL_api_python_unittest.py

```diff
@@ -82,8 +83,8 @@ def main():
     random_seed = args.seed
     local_rank = args.local_rank
     world_size = args.world_size
-    interpolation_type = INTERPOLATION_TYPES[args.interpolation_type]
-    scaling_mode = SCALING_MODES[args.scaling_mode]
+    interpolation_type = INTERPOLATION_TYPES[args.resize_interpolation_type]
```

Comment:
Interpolation type can be used for warp affine and rotate too. Is it necessary to mention resize?

Reply:
Swetha commented in the pybind PR to change it as resize-interpolation type